data/LIKE/data.json

```diff
@@ -1,22 +1,17 @@
 {
   "name": "LikeButton.eth",
-  "symbol": "LIKE",

```

Comment:
I believe this needs to remain `LIKE` so that it matches the folder name

Reply:
Looks like #822 was merged.

I changed the symbol back to “LIKE” to match the repo folder and added the overrides back in as “❤️” for base and ethereum to match the contract symbols.  

Please let me know if this isn’t correct

Reply:
@ENSpunks Thanks!

@hamdiallam Looks like the CI is still failing because of the heart emoji. 🤔